camera_name: ""
image_height: 576
image_width: 928
camera_matrix:
  rows: 3
  cols: 3
  data:
    - 361.51917037736723
    - 0
    - 463.440734517615
    - 0
    - 400.11735811713135
    - 274.15605302450348
    - 0
    - 0
    - 1
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data:
    - -0.22865849153284709
    - 0.049323509522197738
    - -0.00031197617910773737
    - 0.0025543262496354868
    - -0.0044897081925468719
rectification_matrix:
  rows: 3
  cols: 3
  data:
    - 1
    - 0
    - 0
    - 0
    - 1
    - 0
    - 0
    - 0
    - 1
projection_matrix:
  rows: 3
  cols: 4
  data:
    - 361.51917037736723
    - 0
    - 463.440734517615
    - 0
    - 0
    - 400.11735811713135
    - 274.15605302450348
    - 0
    - 0
    - 0
    - 1
    - 0